### PR TITLE
style: improve code quality by refactoring and formatting

### PR DESCRIPTION
**Pull request by himanshu010**
### Description

Fixes # [ISSUE]
N/A
### Type of Change:
code quality improvement

- Code
- Quality Assurance
- User Interface
- Outreach
- Documentation

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged
- [ ] Update Postman API at /docs folder
- [ ] Update Swagger documentation and the exported file at /docs folder
- [ ] Update requirements.txt

**Code/Quality Assurance Only**

- [x] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

### CODE REVIEW

**Comment by robotjellyzone** [comment]
Hi @himanshu010 did you created an issue before sending this PR? If yes, then can you please add that issue number under **fixes** in the description of this PR .? Also, do fill out the **How This has been this Tested**

File: app/api/dao/user.py [hunk 1]
```diff
@@ -144,7 +150,7 @@ def get_user_by_username(username: str):
         return UserModel.find_by_username(username)
 
     @staticmethod
-    def list_users(user_id: int, search_query: str = "", is_verified = None):
+    def list_users(user_id: int, search_query: str = "", is_verified=None):
```
Comment (robotjellyzone) [inline app/api/dao/user.py:153]: Hmm. you don't need to remove that space between = & None

File: app/api/resources/mentorship_relation.py [hunk 2]
```diff
@@ -707,7 +721,9 @@ class CreateTaskComment(Resource):
     @mentorship_relation_ns.expect(auth_header_parser, task_comment_model)
     @mentorship_relation_ns.doc(
         responses={
-            HTTPStatus.CREATED: messages.TASK_COMMENT_WAS_CREATED_SUCCESSFULLY["message"],
+            HTTPStatus.CREATED: messages.TASK_COMMENT_WAS_CREATED_SUCCESSFULLY[
+                "message"
+            ],
```
Comment (robotjellyzone) [inline app/api/resources/mentorship_relation.py:726]: I think this was also not needed ?

Comment (himanshu010) [reply app/api/resources/mentorship_relation.py:726]: @robotjellyzone sorry for late reply, I haven't seen this notification. This formatting is not done manually. It is done automatically according to global standard using black formatter. Closing this.

**Pull request closed without merging**